acts.sh

```diff
@@ -1,5 +1,5 @@
 package: ACTS
-version: v23.4.0
+version: "main"
```

Comment:
main is a "moving" version and it is easy to break the CI when you commit to ACTS main. In general, it would be preferred to use fixed tags (even if they are in your development branch). @plariono : Would you be able to tag your own development in ACTS (with some ALICE specific tag)?

Reply:
Hi @sawenzel, here the "main" branch refers to the AliceO2Group fork, which we use as a temporary solution. 
When the necessary PR gets merged in the main ACTS repository, we'll come back to a fixed version. 
> @plariono : Would you be able to tag your own development in ACTS (with some ALICE specific tag)?
I'm not sure. Where exactly such tag should be?

Reply:
> main is a "moving" version and it is easy to break the CI when you commit to ACTS main. In general, it would be preferred to use fixed tags (even if they are in your development branch). @plariono : Would you be able to tag your own development in ACTS (with some ALICE specific tag)?

Isn't "main" a moving version only if we update the repository? Just to be sure there is not some automatic synchronisation mechanism I am not aware of.

Concerning the tagging, this we can surely do on AliceO2Group/ACTS side.

Reply:
yes sure. but update of the repo might happen without testing. I am not strictly against using "main" ... just wanted to raise the issue.

Reply:
> We would create a tag, by releasing the ACTS code, in the fork of ACTS that you maintain

I see, so it will be a tag in our fork of ACTS, indeed that would be nice.